engine/classes/Elgg/Application/CacheHandler.php

```diff
@@ -83,7 +83,7 @@ public function handleRequest($path) {
 			exit;
 		}
 
-		$filename = $config->getVolatile('dataroot') . 'views_simplecache/' . md5("$viewtype|$view");
```

Comment:
Good point. This was probably a vestige of my experience caching for Minify, as Minify tries hard to name cache files so that you could recognize the files in them. Weird I know.